internal/support/notifications/normal_distribution.go

```diff
@@ -20,12 +20,10 @@ import (
 )
 
 func distributeAndMark(n models.Notification) error {
-	err := distribute(n)
-	if err != nil {
-		LoggingClient.Error("Trouble on distribution of notification: " + n.Slug)
-		return err
-	}
-	err = dbClient.MarkNotificationProcessed(n)
+	go distribute(n)
+	LoggingClient.Info("Request to start distributing notification has been triggered for: " + n.Slug)
```

Comment:
Please remove this line. We don't need this particular log statement. Consider devices with a minimum of disk space. We don't want to fill up the disk or db.

Reply:
how about using Debug instead?